documentation/generating-typedoc.md

```diff
@@ -1,8 +1,8 @@
-# AvalancheJS -  Typedoc Documentation
+# CaminoJS -  Typedoc Documentation
 
 ## Overview
 
-We use [typedoc](https://typedoc.org) to turn comments in our typescript into markdown pages which can be copied into [the Avalanche Docs repo](https://github.com/ava-labs/avalanche-docs) and [the AvalancheJS Docs repo](https://github.com/ava-labs/avalanchejs-docs). Each time a new version of AvalancheJS gets deployed to [npm](https://www.npmjs.com/package/avalanche) then typedocs should be regenerated and pasted into the both docs repo previously mentioned.
+We use [typedoc](https://typedoc.org) to turn comments in our typescript into markdown pages which can be copied into [the CaminoJS Docs repo](https://github.com/chain4travel/caminojs-docs). Each time a new version of CaminoJS gets deployed to [npm](https://www.npmjs.com/package/@c4tplatform/caminojs) then typedocs should be regenerated and pasted into the both docs repo previously mentioned.
```

Comment:
[Camino Docs](https://github.com/chain4travel/caminojs-docs) doesn't exist either change this or we should change the name of the repo [CaminoJs Docs](https://github.com/chain4travel/camino-docs)